footer.html

```diff
@@ -112,7 +112,8 @@
     </ul>
   </div>
   <div class="copyright">
-    <p>© The New York Public Library, 2018</p>
+    <p>© The New York Public Library, 2019</p>
```

Comment:
Ha this should maybe be js so it can update automatically.

Reply:
yeah! I can do that.

Reply:
I would not bother with that change in case this gets updated this year along with the website redesign.

Reply:
ha OK then!